core/templates/services/playthrough-issues.service.ts

```diff
@@ -52,11 +52,10 @@ angular.module('oppia').factory('PlaythroughIssuesService', [
         stateName + '" many times.';
     };
 
-    var renderEarlyQuitIssueSuggestions = function(issue) {
+    var renderEarlyQuitIssueSuggestions = function(stateName) {
```

Comment:
I'm surprised there aren't any other changes that need updating after this change, can you please confirm whether all usages of these functions have been updated (a simple grep should do)?